openrtb_ext/bidders.go

```diff
@@ -115,116 +128,130 @@ const (
 	BidderZeroClickFraud   BidderName = "zeroclickfraud"
 )
 
-// BidderMap stores all the valid OpenRTB 2.x Bidders in the project. This map *must not* be mutated.
-// The bidder name 'general' is not allowed since it has special meaning in message maps.
-var BidderMap = map[string]BidderName{
```

Comment:
This is just a 1:1 mapping. No need to have new adapters add a line here. It's now automatically generated. It's also no longer shares memory, so the `This map *must not* be mutated.` note no longer applies.